eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "ecmaVersion": 2021,
+    "ecmaVersion": 2022,
     "ecmaFeatures": {
       "jsx": true
     },
```

Comment:
Shouldn't we just remove the `parserOptions` completely? This module should always be used in tandem with `eslint-config-standard` or similar, and that one should be responsible for setting these things?